topics/ajax/index.md

```diff
@@ -0,0 +1,9 @@
+---
+display_name: Ajax
+short_description: Ajax is a web technology used by web apps and web pages to exchange data in the background.
+topic: ajax
+related: json, html, xml, javascript
+wikipedia_url: https://en.wikipedia.org/wiki/Ajax_(programming)
+---
+AJAX stands for asynchronous javascript and xml. It is collection of several web technologies such as HTML, CSS, JSON, XML, JavaScript. It is used for creating dynamic webpages in which small part of web page changes without reloading the page. AJAX is used by Gmail and Google maps.
```

Comment:
- `javascript` needs to be changed to `JavaScript`.
- `xml` needs to be changed to `XML`.
- `such as` should probably be changed to `including`, since "such as" implies that your list is an example, whereas "including" implies that the elements in the list are actually a part of the set.
- `AJAX` should be `Ajax` instead since 1) this appears to be the proper spelling now and 2) you need to be consistent with what you used earlier. [Source](https://snook.ca/archives/javascript/ajax_vs_ajax).
- `AJAX is used by Gmail and Google maps` should be removed since 1) this does not describe what Ajax is and 2) the decision of which website to promote here is opinionated at best.

Reply:
I think  display_name should be  AJAX instead of Ajax for consistency.
AJAX seems to be more appropriate 
https://www.w3schools.com/php/php_ajax_intro.asp
https://stackoverflow.com/questions/1510011/how-does-ajax-work
https://www.tutorialspoint.com/ajax/what_is_ajax.htm

Please correct me if I'm wrong.

Reply:
@himanshu1603 I agree with @GloverDonovan: please use "Ajax". Note this is sometimes a thing in English where acronyms that are said (e.g. "Eh-jax") rather than spelt ("Eh-Jay-Eh-Ex") often evolve into being a noun rather than acronym. Thanks!